Update connectivity-settings.md (#84753)
Azure/azure-content: articles/synapse-analytics/security/connectivity-settings.md

ms.topic: overview
ms.subservice: security 

> ms.date: 08/05/2021 

Reviewer: ```suggestion
ms.date: 12/03/2021 
```

author: alexshinmsft 
ms.author: seshin 

> ms.reviewer: jrasnick, wiassaf

Reviewer: ```suggestion
ms.reviewer: wiassaf
```

## Minimal TLS version
Synapse SQL in Azure Synapse Analytics allows connections using all TLS versions. You cannot set the minimal TLS version for Synapse SQL in Azure Synapse Analytics.

> We are in process of setting minimum requirement of TLS1.2 in some regions when you create new Synapse Workspace. In these regions, login attempts from customer who are using a TSL version lower than 1.2 to the newly created Synapse workspace will fail. 

Reviewer: ```suggestion
A requirement for TLS 1.2 has been implemented in some regions, for new Synapse Workspaces only. In these regions, login attempts to the newly created Synapse workspace from connections using a TSL version lower than 1.2 will fail. A list of the regions enforcing TLS 1.2 for new Synapse Workspaces is below:

| Azure Region |
|:---|
| Central US | 
```

Can we add specific regions? Central US is just a sample...
